client/menu/menu.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+    <link rel="stylesheet" href="../../third_party/codemirror-5.19.0/lib/material.css">
+    <link rel="stylesheet" href="menu.css">
+    <script src="../../third_party/angular-1.6.1/angular.min.js"></script>
+  </head>
+  <body>
+    <div class="tie-menu-question-list-wrapper">
```

Comment:
I'm agnostic to that; it's fine to not use a JS file or the assets for now. It's fine to do something simple now as a placeholder, as long as it makes sense and points to the question page (in fact just one link overall is probably better given that we're still showing the same question page regardless of which is clicked).